### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by kosamari**
Fixes few terminology error
**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/09/inside-browser-part2.md [hunk 1]
```diff
@@ -114,10 +114,13 @@ they need to pass the data to download manager.
 
 This is also where the [SafeBrowsing](https://safebrowsing.google.com/) check happens. 
 If the domain and the response data seems to match a known malicious site, then the network thread 
-alerts to display a warning page. Additionally, 
+alerts to display a warning page. 
+
+Note: this is a check point every network request (both page navigation and subresource download)
```
Comment (kosamari) [inline src/content/en/updates/2018/09/inside-browser-part2.md]: @ericlaw1979 I edited the part about CORB per your comment on the post, WDYT ?

Comment (ericlaw1979) [reply src/content/en/updates/2018/09/inside-browser-part2.md]: Grammatically, I think a word is missing. Maybe:

"Note: this is a **checkpoint on** every network request (both page navigation and subresource download)"?

Comment (jpmedley) [reply src/content/en/updates/2018/09/inside-browser-part2.md]: +1

File: src/content/en/updates/2018/09/inside-browser-part2.md [hunk 2]
```diff
@@ -114,10 +114,13 @@ they need to pass the data to download manager.
...
 If the domain and the response data seems to match a known malicious site, then the network thread 
-alerts to display a warning page. Additionally, 
+alerts to display a warning page. 
+
+Note: this is a check point every network request (both page navigation and subresource download) 
+goes through. For subresource download, 
 [**C**ross **O**rigin **R**ead **B**locking (**CORB**)](https://www.chromium.org/Home/chromium-security/corb-for-developers) 
-check happens in order to make sure sensitive cross-site 
-data does not make it to the renderer process.
+check happens here in order to make sure sensitive cross-site data does not make it to the renderer
```
Comment (jpmedley) [inline src/content/en/updates/2018/09/inside-browser-part2.md]: Needs to be 'a check happens...'.

Delete 'in order'. It doesn't contribute any meaning.

**Comment by PaulKinlan** [comment]
Actually, I approved this and I still think there are fixes required... 

Also I fixed a merge issue.

**Comment by petele** [comment]
@kosamari - can you address please.

**Comment by petele** [comment]
@kosamari - any updates on this?

**Comment by petele** [comment]
@kosamari - can you look at this please?

**Comment by petele** [comment]
Any update on this?

**Pull request closed without merging**